deploy/sre-build-test/50-build-test.CronJob.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: sre-build-test
+  namespace: openshift-build-test
+spec:
+  failedJobsHistoryLimit: 5
+  successfulJobsHistoryLimit: 3
+  concurrencyPolicy: Forbid
+  schedule: "11 */1 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          affinity:
+            nodeAffinity:
+              requiredDuringSchedulingIgnoredDuringExecution:
+                nodeSelectorTerms:
+                - matchExpressions:
+                  - key: "node-role.kubernetes.io"
+                    operator: In
+                    values:
+                      - infra
+          tolerations:
+          - operator: Equal
+            key: node-role.kubernetes.io
+            value: infra
+            effect: NoSchedule
+          serviceAccountName: sre-build-test
+          restartPolicy: Never
+          containers:
+          - name: sre-build-test
+            image: image-registry.openshift-image-registry.svc:5000/openshift/cli:latest
+            imagePullPolicy: Always
+            command:
+            - /bin/sh
+            - -ec
+            - "export NS=openshift-build-test-$(date +%s) ; oc create ns $NS && oc -n $NS new-build nodejs~https://github.com/openshift/nodejs-ex --name=sre-build-test; echo $(date): Waiting for build to complete.; while :; do oc -n $NS get build -o custom-columns=STATUS:.status.phase --no-headers | grep Complete && break; echo $(date): Checking build again in 15 seconds, NS=$NS; sleep 15; done; echo $(date): Done, deleting build NS=$NS; oc delete ns $NS"
```

Comment:
I think it would be more readable if this was contained in a proper shell script, perhaps mounted into the container via a ConfigMap.

Also, I think the node build uses a lot of resources (ISTR a discussion around the resource usage) and that a Ruby-based one might be better suited?

Reply:
Is this request for a CM a blocker to merge?

Re resources, happy to pick something else but am just basing this on what we do for all OSD v3.. not concerned w/ the resource utilization.

Reply:
It's a blocker in the sense that reviewing a 400+ character one-liner is hard to review. Splitting it into a script lets us get away from such a long line, and grant readability. Yes, I could cut and paste it into an editor and turn it into a multi-liner, but future changes to this will also be challenging to do.

Reply:
Updated to use `>-` so it's broken into multiple lines in the repo.  When it's deployed to the cluster it's munged together again with `\n` added in, but this makes it easier to reason about in code.